README.md

```diff
@@ -3,7 +3,7 @@
 
 [![CircleCI Build Status](https://circleci.com/gh/electron/electron/tree/master.svg?style=shield)](https://circleci.com/gh/electron/electron/tree/master)
 [![AppVeyor Build Status](https://windows-ci.electronjs.org/api/projects/status/nilyf07hcef14dvj/branch/master?svg=true)](https://windows-ci.electronjs.org/project/AppVeyor/electron/branch/master)
-[![Azure Pipelines Build Status](https://github.visualstudio.com/electron/_apis/build/status/electron-builds/electron-osx-testing?branchName=master)](https://github.visualstudio.com/electron/_build/latest?definitionId=36)
+[![Azure Pipelines Build Status](https://dev.azure.com/github/electron/_apis/build/status/electron-builds/electron-osx-testing?branchName=master)](https://github.visualstudio.com/electron/_build/latest?definitionId=36)
```

Comment:
@chrisrpatterson are all the URLS changing for Azure Devops?  If so, should we also change the reference to `https://github.visualstudio.com/electron/_build/latest?definitionId=36`?

Reply:
@jkleinsc I think they'll support both for a while, but it does make sense to rename that URL too while we're in here